libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -678,25 +678,13 @@ bool PbftManager::stateOperations_() {
 
   // Periodically verify unverified votes
   vote_mgr_->verifyVotes(round, [this](auto const &v) {
-    auto dpos_votes_count = dposEligibleVoteCount_(v->getVoterAddr());
-    if (!dpos_votes_count) return false;
```

Comment:
if you remove this we will have a lot of warning msg now, as before `LOG(log_wr_) << e.what();` was never executed